### PR TITLE
Update readme with setup instructions

### PR DESCRIPTION
**Pull request by thomas-brunton**

### CODE REVIEW

File: frontend/bonds-for-success/src/app/libs/api/contracts/api.yaml [hunk 1]
```diff
@@ -43,13 +43,43 @@ paths:
                 type: array
```
Comment (Tsz-Fung-Ryan) [inline frontend/bonds-for-success/src/app/libs/api/contracts/api.yaml]: Need to update the open api to latest cut I made

**Review by Tsz-Fung-Ryan — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -4,3 +4,46 @@ FullStack Matching Software for CIC
...
+
+  How to Update the Backend
+    Updating the properties (an installation of maven is required to regenerate the jar)
+      1. Navigate to backend/application.properties
+      2. Under mentee.key.mappings update the properties the following way:
+        a. Write the name of the column then a colon followed by what to map the property to
+          ex: "What is your name?": 'name'
+        b. current sourced properties are: name, emailAddress, genderPreference, gender, keywords
+      3. Repeat step 2 but for mentor.key.mappings using the same structure
+      4. To update the keywords property merely add or remove the indicated keepwords with each one seperated by a comma
```
Comment (thomas-brunton) [inline README.md]: I think you meant to write `indicated keywords`?

File: README.md [hunk 3]
```diff
@@ -4,3 +4,46 @@ FullStack Matching Software for CIC
...
+      5. The accepted threshold is the percentage of keywords the program will look for to be considered an "ideal" match do note that with a higher number the later matches will be less optimal (the logs will indicate if an ideal match could not be done with the remainder matches)
```
Comment (thomas-brunton) [inline README.md]: Add a period after `"ideal" match`

File: README.md [hunk 4]
```diff
@@ -4,3 +4,46 @@ FullStack Matching Software for CIC
...
+  Download the project
+    1. Navigate to https://github.com/Tsz-Fung-Ryan/BondsForSuccess in a web browser
+    2. Click the green `Code` button
+    3. Click the `Download ZIP` button
+    4. Unzip the downloaded zip file somewhere
+
+  Running the backend
+    1. Install Java 18: https://www.oracle.com/java/technologies/downloads/
+    2. Head to the backend folder
+    3. Double click on the jar to turn on the backend
```
Comment (thomas-brunton) [inline README.md]: Maybe say the name of the file that they are supposed to click on? Not sure if they would know what a jar file is

File: README.md (hunk 2 above)
Comment (Tsz-Fung-Ryan) [reply README.md]: done

**Pull request merged**